fs/littlefs/Make.defs

```diff
@@ -50,6 +50,9 @@ $(LITTLEFS_TARBALL):
 .littlefsunpack: $(LITTLEFS_TARBALL)
 	$(Q) tar zxf littlefs/$(LITTLEFS_TARBALL) -C littlefs
 	$(Q) mv littlefs/littlefs-$(LITTLEFS_VERSION) littlefs/littlefs
+ifeq ($(CONFIG_ARCH_USE_MMU),y)
```

Comment:
Maybe the patch should be applied unconditionally?

Reply:
Yes, I think so.

Reply:
@radek-pesina why mark this issue as resolved?

Reply:
Apologies!  I pushed up without adding the file :S